README.md

```diff
@@ -27,6 +27,9 @@
 -->
 
 [![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+![Static Badge](https://img.shields.io/badge/Triton-24.08-8A2BE2)
+![Static Badge](https://img.shields.io/badge/vLLM-0.5.5-blue)
```

Comment:
could you please clarify why the version 0.5.5 was picked?

Reply:
We have a passing pipeline with this version. Hence, I picked this as the default until a new version is tested and verified.

Reply:
I think we should have the latest version we support, or wait until we have tests, that indicate that we can migrate to the latest. Since we don't support 0.5.5 and vLLM's latest version now is 0.6.1.post2, I can see a confusion, that will arise in users with this badge

Reply:
done